src/components/SetupStrategyBasic.vue

```diff
@@ -11,7 +11,7 @@ const DEFAULT_TOKEN = {
   logo: '',
   standard: 'ERC-20',
   symbol: '',
-  decimals: null
+  decimals: '18'
```

Comment:
If the token doesnt load it would give 18 decimals? That could be problematic for some token not haivng 18 decimals